### PR TITLE
feat(locales): add czech locale

### PR DESCRIPTION
**Pull request by nwvh**
Added and translated cs.json

### CODE REVIEW

**Review by TheFantomas — APPROVED** [review]

File: locales/cs.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "empty_slot": "Prázdný slot",
+  "create_character": "Vytvořit novou postavu",
+  "select_character_title": "Výběr Postavy",
+  "delete_character_title": "Smazat Postavu",
+  "select_character": "Vyber si svou postavu",
```
Comment (ESK0) [inline locales/cs.json:6]: Zvolit postavu

File: locales/cs.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "empty_slot": "Prázdný slot",
+  "create_character": "Vytvořit novou postavu",
+  "select_character_title": "Výběr Postavy",
```
Comment (ESK0) [inline locales/cs.json:4]: No need to second word begin with a capital letter

File: locales/cs.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  "delete_character_title": "Smazat Postavu",
```
Comment (ESK0) [inline locales/cs.json:5]: No need to second word begin with a capital letter

File: locales/cs.json (hunk 1 above)
Comment (nwvh) [reply locales/cs.json]: Thank your for spotting these mistakes, I have fixed them, but I think "Vybrat postavu" sounds better than "Zvolit postavu" as you suggested.

File: locales/cs.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{
+  "empty_slot": "Prázdný slot",
+  "create_character": "Vytvořit novou postavu",
+  "select_character_title": "Výběr postavy",
+  "delete_character_title": "Smazání postavy",
+  "select_character": "Vybrat postavu",
+  "delete_character": "Smazat postavu",
+  "delete_character_confirm": "Jsi si jistý(á), že chceš smazat postavu %s %s?  \nTuto akci nelze vrátit.",
```
[the remark this answers is not in the document]
Comment (ESK0) [reply locales/cs.json:8]: > This is subjective, but using `jistý(á)` looks weird and in these days it also doesn't represent all genders properly 🙄 I would suggest using something like "Jste si jistí, že chcete ..." or "Opravdu chcete smazat ...".

What you mean all genders?

[the remark this answers is not in the document]
Comment (nwvh) [reply locales/cs.json:8]: 🌚

**Pull request closed without merging**